blackboxopt/optimization_loops/dask_distributed.py

```diff
@@ -102,6 +105,7 @@ def run_optimization_loop(
     dask_client: dd.Client,
     timeout_s: float = float("inf"),
     max_evaluations: int = None,
+    exit_on_unhandled_exception: bool = False,
```

Comment:
We really want `False` as default? At least I usually run earlier into the issue, that I miss the exceptions, where `True` would be helpful. If I remember correctly, this was also sometimes the case for the users. But I also don't know, how common those "numerical instabilities" are...